docs/restrict-dynamic-log-locations.md

```diff
@@ -23,13 +23,95 @@ The benefits of restricting dynamic log locations in MySQL outweigh the disadvan
 
 ## secure_log_path
 
-The variable is read-only and must be set up in a configuration file or the command line.
+| Variable Name | Description       |
+|---------------|-------------------|
+| Command-line  | --secure-log-path |
+| Dynamic       | No                |
+| Scope         | Global            |
+| Data type     | String            |
+| Default       | empty string      |
+
+This variable restricts the location of the `slow_query_log`, `general_log`, and `buffered_error_log` files. Also, this variable is applied to the following options:
+
+* `slow_query_log_file` - specifies the name of the slow query log file and the directory where to store this file.
+
+* `general_log_file` - specifies the name of the general log file and the directory where to store this file.
+
+* `buffered_error_log_filename` - specifies the name of the buffered error log file and the directory where to store this file. You can specify the size of the buffer for error logging in bytes with the `buffered-error-log-size` option.
```

Comment:
You can specify the size of the buffer for error logging in bytes with the `buffered-error-log-size` option.

Why is this mentioned?

Reply:
The buffered-error-log-size option is mentioned in the example below. That's why I explained it here.

Reply:
Then, make that variable another item on the list. It's kind of shoehorned into this variable's description.

Reply:
added